### PR TITLE
Port collection of minor changes

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/245/files
https://github.com/PureStake/moonbeam-docs/pull/241/files
https://github.com/PureStake/moonbeam-docs/compare/396c9a9abf3f793308ecbf4b5d201d643681b0e8...353558519f71d14be9d1ff242998228bdef054e6

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/integrations/indexers/thegraph.md [hunk 1]
```diff
@@ -233,35 +231,33 @@ export function handlePlayerJoined(event: PlayerJoined): void {
     player = new Player(playerId);
   }
   player.address = event.params.player;
-
   player.save();
-
   // ID for the ticket (round - player_address - ticket_index_round):
   // "round_number" + "-" + "player_address" + "-" + "ticket_index_per_round"
```
Comment (eshaben) [inline builders/integrations/indexers/thegraph.md:236]: this line can be deleted

File: builders/integrations/indexers/thegraph.md [hunk 2]
```diff
@@ -233,35 +231,33 @@ export function handlePlayerJoined(event: PlayerJoined): void {
...
+  // `${round_number}-${player_address}-${ticket_index_per_round}`
   let nextTicketIndex = event.params.ticketIndex.toString();
   let ticketId = roundId + "-" + playerId + "-" + nextTicketIndex;
```
Comment (eshaben) [inline builders/integrations/indexers/thegraph.md:239]: this line can also be deleted

File: builders/tools/eth-dev-env/openzeppelin/defender.md [hunk 3]
```diff
@@ -107,13 +107,13 @@ contract PausableBox is Initializable, PausableUpgradeable, OwnableUpgradeable {
...
 - **Admin action** —— 可调用受管理合约的任何函数
 
 在本示例中，我们创建了一个新提案，提案内容为暂停合约。为此，需要进行以下操作：
 
- 1. 点击“New proposal（新建提案）”按钮，显示所有选项
- 2. 点击“Pause（暂停）”
+ 1. 点击**New proposal**（新建提案）按钮，显示所有选项
+ 2. 点击**Pause**（暂停）
 
 ![OpenZeppelin Defender Admin Contract New Pause Proposal](/images/builders/tools/eth-dev-env/openzeppelin/oz-defender-4.png)
```
Comment (eshaben) [inline builders/tools/eth-dev-env/openzeppelin/defender.md:118]: this was as close as I could get to the actual link haha but line 125 should be updated to **Create pause proposal**

File: builders/tools/multisig-safe.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ contract SetText {
 
 ## 创建一个Safe钱包 {: #create-a-safe }
 
-要创建一个Safe钱包，导航至[Moonbeam Safe](https://multisig.moonbeam.network/moonbase)。本教程将引导您在Moonbase Alpha上创建一个Safe钱包，您也可以修改配置[在Moonriver上创建一个Safe钱包](https://multisig.moonbeam.network/moonriver)。您可点击页面右上角的网络下拉菜单切换网络。
+要创建一个Safe钱包，导航至[Moonbeam Safe](https://multisig.moonbeam.network/moonbase){target=blank}。本教程将引导您在Moonbase Alpha上创建一个Safe钱包，您也可以修改配置在[Moonbeam](https://multisig.moonbeam.network/moonbeam){target=blank}或[Moonriver](https://multisig.moonbeam.network/moonriver){target=blank}上创建一个Safe钱包。您可点击页面右上角的网络下拉菜单切换网络。
 
 ### 连接MetaMask {: #connect-metamask }
```
Comment (eshaben) [inline builders/tools/multisig-safe.md:70]: the line below this one is missing {target=blank}

**Review by eshaben — CHANGES_REQUESTED** [review]

File: builders/integrations/indexers/thegraph.md (hunk 1 above)
Comment (eshaben) [reply builders/integrations/indexers/thegraph.md:236]: this is displaying weird, but this line is the one that can be deleted `  // "round_number" + "-" + "player_address" + "-" + "ticket_index_per_round"`

File: builders/integrations/indexers/thegraph.md (hunk 2 above)
Comment (eshaben) [reply builders/integrations/indexers/thegraph.md:239]: this line `  let ticketId = roundId + "-" + playerId + "-" + nextTicketIndex;`

**Comment by hyd628** [comment]
Thanks, feedback incorporated.

**Review by eshaben — APPROVED** [review]
LGTM

**Pull request merged**